front/src/applications/stdcm/components/StdcmResults/StdcmResults.tsx

```diff
@@ -93,6 +98,57 @@ const StdcmResults = ({
     return extractMarkersInfo(outputs.results.simulationPathSteps);
   }, [hasSimulationResults, outputs]);
 
+  const [refSchedule, setRefSchedules] = useState<RefSchedule[]>([]);
+
+  const [postRefSchedules] = osrdEditoastApi.endpoints.postSimilarTrains.useMutation();
+  const requestTrainSchedules = async () => {
+    const { consist, pathSteps } = selectedSimulation.inputs;
+    if (!consist) {
+      return;
+    }
+
+    const key = (uic: number, ch: string) => `${uic}-${ch}`;
+
+    const stops = pathSteps.reduce((acc, ps) => {
```

Comment:
`stops` is not a very clear naming.

I'm expected an array of stops, but actually it's a map which indicates for each op whether it has a stop or not

Reply:
done